lora_verifier/src/entropy.rs

```diff
@@ -15,6 +15,7 @@ pub struct Entropy {
     pub id: Vec<u8>,
     pub data: Vec<u8>,
     pub timestamp: DateTime<Utc>,
+    pub version: i32,
```

Comment:
does this want to be a `u32`? the proto version seems to be a `uint32`

Reply:
or is this a limitation of storing the value in postgres?

Reply:
so.... the gateway repo has a `beacon` crate as part of the beacon PR that defines this exact structure We should endeavor to use that instead

Reply:
the reason for i32 here is that postgres wont accept a uint